Submodels/MTFLO_call.py

```
@@ -1,20 +1,185 @@
+        if self.process.poll() is None:
+            try:
+                self.process.wait(timeout=2)
+            
+            except subprocess.TimeoutExpired:
+                self.process.kill()
+                raise OSError("MTFLO did not close after file generation. Process was killed.") from None
+        else:    
+            return 
+
+
+    def caller(self
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)